tensorflow/python/ops/math_ops.py

```diff
@@ -2786,7 +2786,19 @@ def add_n(inputs, name=None):
   """Adds all input tensors element-wise.
 
   Converts `IndexedSlices` objects into dense tensors prior to adding.
-
+  
+  `tf.math.add_n` performs the same operation as `tf.math.accumulate_n` but it waits
+  for all of its inputs to be ready before beginning to sum, that consumes more 
+  memory as  inputs are not ready at different times, since minimum temporary storage 
+  is proportional to the output size rather than the inputs size.
+  For example:
```

Comment:
I'd suggest putting a newline before "For example:" otherwise the documentation generator will put the "For example:" at the end of the previous sentence.

For reference you can use gist.github.com to see how your markdown will end up get rendered. Here is the current version: https://gist.github.com/tomhennigan/d2789426d38bdbd9471e7e00555eced0